lib/mastodon/media_cli.rb

```diff
@@ -113,5 +113,24 @@ def usage
       say("Imports:\t#{number_to_human_size(Import.sum(:data_file_size))}")
       say("Settings:\t#{number_to_human_size(SiteUpload.sum(:file_file_size))}")
     end
+
+    desc 'lookup', 'Lookup a toot by passing a media URL'
+    def lookup
+      prompt = TTY::Prompt.new
+
+      exit(1) unless (url = prompt.ask('Please enter a URL to the media to lookup:', required: true))
+
+      id = url
+           .split('/')[0..-2]
```

Comment:
do we know the media url is always mounted on root? is there a way to configure s3 with a prefix?

Reply:
We don't know it's mounted on root. This check is folder agnostic. It decides the URL into parts and pulls the parts with numbers only to determine an attachment ID. To the best of my knowledge whether it's mounted to root of the hostname or not does not affect this check.